S3/S3.py

```diff
@@ -481,6 +481,7 @@ def bucket_info(self, uri):
             response['requester-pays'] = self.get_bucket_requester_pays(uri)
         except S3Error as e:
             response['requester-pays'] = None
+        response['versioning'] = self.get_versioning(uri)
```

Comment:
I think that you can put a try/except around that also, as I easily guess that not all servers support versioning